lessons/codap-bar-charts/langs/en-us/pages/plot-practice-3.adoc

```diff
@@ -99,12 +99,12 @@ The display shows us the relationship between animals' age and weight.
 
 |
 @ifnotsoln{
-* [ ] Quantitative
+* [ ] Numeric
 * [ ] Categorical
 }
 
 @ifsoln{
-* [x] Quantitative
+* [x] Numeric
```

Comment:
I'm curious how you cam up with the checkboxes here. I was expecting them to mirror what I see in the dropdown menu, but box plot and least squares line never appear simultaneously in the drop down menus... also, I would have assumed that students study box plots before studying regression (which is to say I would have prioritized the box plot over the least squares line).